Symfony/CS/ConfigInterface.php

```diff
@@ -105,4 +105,20 @@ public function usingCache();
      * @return bool
      */
     public function usingLinter();
+
+    /**
+     * Sets the path to the cache file.
+     *
+     * @param string $cacheFile The path to the cache file
+     *
+     * @return ConfigInterface The same instance
```

Comment:
@GrahamCampbell 

The implementation of `addCustomFixer()` in `Config` is chainable, just as any other mutator in `Config`. However, I removed it from `ConfigInterface`.

I suggest to keep it consistent, though.